tests/unit/factories.py

```diff
@@ -50,58 +69,69 @@ class Meta:
 
 
 class MockGithubRunnerCharmUnitFactory(factory.Factory):
+    """Mock github-runner charm unit."""  # noqa: DCO060
+
     class Meta:
+        """Configuration for factory."""  # noqa: DCO060
+
         model = MagicMock
 
     name = "github-runner/0"
 
 
 class MockGithubRunnerCharmAppFactory(factory.Factory):
-    """Mock github-runner charm app."""
+    """Mock github-runner charm app."""  # noqa: DCO060
 
     class Meta:
+        """Configuration for factory."""  # noqa: DCO060
+
         model = MagicMock
 
     planned_units = 1
     name = "github-runner"
 
 
 class MockGithubRunnerCharmModelFactory(factory.Factory):
-    """Mock github-runner charm model."""
+    """Mock github-runner charm model."""  # noqa: DCO060
 
     class Meta:
+        """Configuration for factory."""  # noqa: DCO060
+
         model = MagicMock
 
     relations: dict[str, list] = {COS_AGENT_INTEGRATION_NAME: [], DEBUG_SSH_INTEGRATION_NAME: []}
 
 
 class MockGithubRunnerCharmFactory(factory.Factory):
-    """Mock GithubRunnerCharm."""
+    """Mock GithubRunnerCharm."""  # noqa: DCO060
+
+    # Docstrings have been abbreviated for factories, checking for docstrings on model attributes
+    # can be skipped.
```

Comment:
Could you please move the explanation of DC060 to the top so that the reader who is not looking at the whole file does not have to look it up? Thank you.